mods/cnc/maps/nod07c/nod07c.lua

```diff
@@ -0,0 +1,204 @@
+NodUnitsVehicles1 = { 'bggy', 'bggy', 'bike', 'bike' }
+NodUnitsVehicles2 = { 'ltnk', 'ltnk' }
+NodUnitsEngineers = { 'e6', 'e6', 'e6', 'e6' }
+NodUnitsRockets = { 'e3', 'e3', 'e3', 'e3' }
+NodUnitsGunners = { 'e1', 'e1', 'e1', 'e1' }
+NodUnitsFlamers = { 'e4', 'e4', 'e4', 'e4' }
+
+Targets = { Actor76, Actor77, Actor78, Actor79, Actor80, Actor81, Actor82, Actor83, Actor84, Actor86, Actor87, Actor88, Actor89, Actor93, Actor94, Actor95, Actor96, Actor97, Actor104, Actor105, Actor106, Actor107, Actor108, Actor157, Actor158, Actor159, Actor160 }
```

Comment:
This would be a good usecase for the newish ScriptTags feature.  You could add a tag to all of the targets in map.yaml (eventually the map editor) and fetch these by the API.